### ISSUE TITLE
pnpm required as of 3.4.0

### ISSUE THREAD

**Issue by dkempner** [magidoc-org/magidoc#193]
### Describe the bug

Fix is to downgrade to 3.3.0

### Reproduction

On a system without pnpm installed

```
npm install -g @magidoc/cli@latest
magidoc generate -f magidoc/magidoc.mjs -p npm
```

### Logs

```shell
> @magidoc/starter-carbon-multi-page@3.4.0 build
> pnpm sync && vite build
sh: 1: pnpm: not found
```

### System Info

```shell
Node v18.13.0
```

### Severity

Blocking an upgrade

**Comment by pelletier197** [comment]
Hi @dkempner, I have noticed the problem this morning and have already release a 3.4.1 :slightly_smiling_face:  Let me know if you have any more problem!

**Comment by dkempner** [comment]
> Hi @dkempner, I have noticed the problem this morning and have already released a 3.4.1 🙂 Let me know if you have any more problem, and sorry for the inconvenience.

Thanks so much! Just tested and everything is good. Have a great day :)
